plenum/server/replica_validator.py

```diff
@@ -52,7 +55,7 @@ def validate_3pc_msg(self, msg):
         if view_no == self.replica.viewNo and node.view_change_in_progress:
             return STASH_VIEW, FUTURE_VIEW
 
-        # If Catchup in View Change finished then process a message
+        # If Catchup in View Change finished then process Commit messages
         if node.is_synced and node.view_change_in_progress:
             return PROCESS, None
```

Comment:
Explicit check that this is a really commit message would be good here.

Reply:
The explicit check is already done above.